Nautilus/Assets/Gadgets/ScanningGadget.cs

```diff
@@ -40,6 +40,16 @@ public class ScanningGadget : Gadget
     /// </summary>
     public TechCategory CategoryForPda { get; set; }
 
+    /// <summary>
+    /// Whether to add after or insert before the target, for sorting purposes.
+    /// </summary>
+    public bool PdaAppendAfter { get; set; }
```

Comment:
Maybe change docs to something along the lines of this:
`Whether the blueprint is added before (false) or after (true) the <see cref="PdaSortTarget"/> in the PDA, when that property is assigned.`
In my mind, PdaAppendAfter = false could imply no sorting done at all. However, that's not the case. I almost think an enum could be fitting here as well... but maybe a bit of bloat for the file?
```csharp
public enum InsertionPosition {
    AfterTarget,
    BeforeTarget,
    Beginning,
    End
}
```
Thoughts? Not really sure myself.

Edit: After looking at it more, I think an enum would definitely clear up some ambiguity and possible mistakes we could make down the line.

Reply:
It sounds good, but would have to be only 2 entries: `AppendAfter` & `InsertBefore` as the code needs to insert it somewhere when the target is not found, and since the target is `TechType.None` by default, it doesn't make sense to have separate Enums for `Beginning` & `End`.

Reply:
Alright, fair. So then it would be AppendAfter by default. Sounds good to me.

Reply:
I accidentally deleted my comment it seems.

Yes, that sounds best.

Reply:
I've added an Enum and some tweaks to documentation.